src/screens/ChatScreen/components/ChatHeader.js

```diff
@@ -1,85 +1,99 @@
 import React, { useRef, useCallback, useMemo } from 'react';
-import {
-  withStyles,
-  Icon,
-  TopNavigation,
-  TopNavigationAction,
-  Spinner,
-} from '@ui-kitten/components';
-import BottomSheetModal from 'components/BottomSheet/BottomSheet';
+import { useTheme } from '@react-navigation/native';
```

Comment:
This component is very big. Can we split it into multiple components?

Reply:
@muhsin-k Sure I can pick it as a separate task. If its fine we can merge this PR. Its open for a while.

Reply:
@muhsin-k Made some changes in reducing code lines. https://github.com/chatwoot/chatwoot-mobile-app/pull/666/commits/1e2f352b9fdef2c86d9f0af871a275b905f218cb
(reduced 100 lines)